File: Backend/UAM-LIFT/src/main/java/ni/edu/uam/UAM_LIFT/enums/TipoNotificacion.java
package ni.edu.uam.UAM_LIFT.enums;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum TipoNotificacion {
    @JsonProperty("CANCELACION_VIAJE")
    CANCELACION_VIAJE,

    @JsonProperty("INICIO_VIAJE")
    INICIO_VIAJE,

    @JsonProperty("FINALIZACION_VIAJE")
    FINALIZACION_VIAJE,

    @JsonProperty("USUARIO_UNIDO")
    USUARIO_UNIDO,

    @JsonProperty("USUARIO_ELIMINADO")
    USUARIO_ELIMINADO,

    @JsonProperty("GENERAL")
    GENERAL
}

File: Uam Lift/app/src/main/java/ni/edu/uam/uamlift/data/enums/TipoNotificacion.kt
package ni.edu.uam.uamlift.data.enums

import com.google.gson.annotations.SerializedName

enum class TipoNotificacion {
    @SerializedName("CANCELACION_VIAJE") CANCELACION_VIAJE,
    @SerializedName("INICIO_VIAJE") INICIO_VIAJE,
    @SerializedName("FINALIZACION_VIAJE") FINALIZACION_VIAJE,
    @SerializedName("USUARIO_UNIDO") USUARIO_UNIDO,
    @SerializedName("USUARIO_ELIMINADO") USUARIO_ELIMINADO,
    @SerializedName("GENERAL") GENERAL
}